frontend/src/pages/Home/components/DataModels/Layouters/utils.js

```diff
@@ -0,0 +1,88 @@
+import { Position, internalsSymbol } from 'reactflow';
+import ResponseError from "../../../../../ResponseError";
+
+
+
+function getHandleCoordsByPosition(node, handlePosition) {
+    // all handles are from type source, that's why we use handleBounds.source here
+    const handle = node[internalsSymbol].handleBounds.source.find(
+        (h) => h.position === handlePosition
```

Comment:
```suggestion
        h => h.position === handlePosition
```
Pas primordial, mais les parenthèses sont inutiles